### ISSUE TITLE
Crashing when closing additional windows on Qt

### ISSUE THREAD

**Issue by NiceneNerd** [r0x0r/pywebview#525]
### Specification
- pywebview version: 3.2
- platform / version: Manjaro Linux 64-bit, using Qt

### Description  
  
When using the Qt renderer, closing additional windows kills the whole app. This is true both of regular addition windows and devtools.
  
### Practicalities

- YES/NO I am willing to work on this issue myself.
Yes
- YES/NO I am prepared to support this issue financially.
No

**Comment by r0x0r** [comment]
I spent an hour investigating this, but it is still a mystery why this occurs. Tested with QT 5.14.2.

Some findings
- It occurs randomly. Sometime application quits, sometimes it continues running. I have tested `examples/multiple_windows.py` and the problem occurs with any window (when it occurs).
- aboutToQuit event is never triggered. No error reports, nothing.

Meanwhile I have fixed a bug preventing a child window being created in a thread, but it is not related to this issue.
